go/vt/vttablet/tabletmanager/vreplication/engine.go

```diff
@@ -185,13 +182,11 @@ func (vre *Engine) Exec(query string) (*sqltypes.Result, error) {
 	}
 	defer dbClient.Close()
 
-	if vre.mustCreate {
-		for _, query := range binlogplayer.CreateVReplicationTable() {
-			if _, err := dbClient.ExecuteFetch(query, 0); err != nil {
-				return nil, err
-			}
+	// Create vreplication table if it's missing. This is a no-op if it already exists.
```

Comment:
I still can't figure out how `vre.mustCreate` flag got reset without the db & table being created. One possibility is that the initial attempt returned a different error than `1146`.

Although this is an uncomfortable fail-safe, I think we have to do this because 'it has happened before'. The tests need fixing because they need to field these new queries.